### PR TITLE
Mainnet specs #234

### PR DESCRIPTION
**Pull request by saraswatpuneet**
# Goal
The goal of this PR is to continue for #234 and 
* Add frequency runtime (this is our mainnet runtime)
* Add chain spec for frequency
* Update scripts and jobs to generate mainnet specs/wasm
* refactor existing local runtime/specs to name it frequency-local
* add **TODOs** placeholder in frequency spec for mainnet to be populated with required configuration 

# Checklist
- [x] Chain spec updated
- [x] Run tests for instant sealing as well as local net for correctness

### CODE REVIEW

File: scripts/generate_specs.sh [hunk 1]
```diff
@@ -30,7 +30,7 @@ case $build_step in
     ;;
   build-testnet)
     echo "Building Spec for frequency as rococo testnet"
-    $PWD/target/release/frequency build-spec --chain=frequency_rococo --disable-default-bootnode > ./res/genesis/testnet/frequency-spec-rococo-testnet.json
+    $PWD/target/release/frequency build-spec --chain=frequency-rococo --disable-default-bootnode > ./res/genesis/testnet/frequency-spec-rococo-testnet.json
```
Comment (wilwade) [inline scripts/generate_specs.sh:33]: I still cannot track when to use `-` and when `_`. So a ping to double check this line.

File: scripts/generate_specs.sh [hunk 2]
```diff
@@ -30,7 +30,7 @@ case $build_step in
     ;;
   build-testnet)
```
Comment (wilwade) [inline scripts/generate_specs.sh:31]: Should we rename this to `build-rococo`?

**Review by wilwade — CHANGES_REQUESTED** [review]
Few small things I found. Really nice to have all these setup.

File: scripts/generate_specs.sh (hunk 1 above)
Comment (saraswatpuneet) [reply scripts/generate_specs.sh:33]: --chain "{spec-name}" triggers this part of cli, so i am going to change to following

1. "dev" | "rococo-2000" |"" i.e. either of these tags will point to local specs
2. "rococo-4044" | "testnet" will point to testnet specs, and 
3 "frequency" is mainnet 

https://github.com/LibertyDSNP/frequency/blob/1d907b077151c0649f3f6ae7ba56e2b290de09f0/node/cli/src/command.rs#L51

**Review by wilwade — APPROVED** [review]
🎉

File: Makefile [hunk 3]
```diff
@@ -35,11 +35,14 @@ install-toolchain:
...
+specs-rococo-2000:
+	./scripts/generate_specs.sh 2000 rococo-2000
 
-specs-testnet:
-	./scripts/generate_specs.sh 4044 build-testnet
+specs-rococo-4044:
+	./scripts/generate_specs.sh 4044 rococo-4044
+
+specs-mainnet:
+	./scripts/generate_specs.sh 999 mainnet
```
[the remark this answers is not in the document]
Comment (saraswatpuneet) [reply Makefile:45]: Good catch @demisx! Will do

**Pull request merged**